### PR TITLE
Display patient flows

### PR DESCRIPTION
**Pull request by LisaManresa**
- [x] Add the Clubhouse Story ID: [ch21609]

### CODE REVIEW

File: src/js/views/patients/patient/dashboard/add-action.scss [hunk 1]
```diff
@@ -7,6 +7,10 @@
     color: color(blue);
   }
 
+  .picklist__flow-item .icon {
```
Comment (paulfalgout) [inline src/js/views/patients/patient/dashboard/add-action.scss]: I think this should be `add-action` prefixed.  `picklist` makes it look like it'd be found in the picklist module scss.

File: src/js/views/patients/patient/dashboard/add-workflow_views.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ import intl from 'js/i18n';
 
 import './add-action.scss';
 
-const ItemTemplate = hbs`{{#if isFar}}{{far iconType}}{{else}}{{fas iconType}}{{/if}} <a>{{ name }}</a>`;
+const ItemTemplate = hbs`<span class="{{#if isFar}}picklist__action-item{{else}}picklist__flow-item{{/if}}">{{#if isFar}}{{far iconType}}{{else}}{{fas iconType}}{{/if}} <a>{{ name }}</a></span>`;
```
Comment (paulfalgout) [inline src/js/views/patients/patient/dashboard/add-workflow_views.js]: I don't like reusing `isFar` here it means "is fontawesome regular" we might want a different check for the type.  Is the __action-item class used?

Comment (paulfalgout) [reply src/js/views/patients/patient/dashboard/add-workflow_views.js]: actually should change even be in this PR?

File: test/fixtures/config/flows.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const faker = require('faker');
+const moment = require('moment');
+
+module.exports = {
+  generate() {
+    return {
+      id: faker.random.uuid(),
+      name: `${ faker.company.bsBuzz() } ${ faker.company.catchPhraseNoun() }`,
+      details: faker.lorem.sentences(),
+      due_date: moment(faker.date.between(
```
Comment (paulfalgout) [inline test/fixtures/config/flows.js]: I don't think flows have `due_date` or `duration` but will have a `created_at`

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/views/patients/patient/dashboard/add-workflow_views.js (hunk 2 above)
Comment (LisaManresa) [reply src/js/views/patients/patient/dashboard/add-workflow_views.js]: I can remove it. Just figured it was easy pickings, but I can leave it for @luketlancaster's PR if that's the more logical place for it

File: test/support/api/flows.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+      return mutator({
+        data: action,
+        included: [],
+      });
+    },
+  })
+    .as('routeFlow');
+
+  // cy.routeProgramByAction();
```
Comment (luketlancaster) [inline test/support/api/flows.js]: this needed?

File: src/js/views/patients/patient/data-events/action-item.hbs [hunk 5]
```diff
@@ -0,0 +1,9 @@
+    <td class="table-list__cell w-40">
+      <span class="patient__action-icon">{{far "file-alt"}}</span>{{ name }}
+    </td>
+    <td class="table-list__cell w-60">
+      <span class="table-list__meta" data-state-region></span>
+      <span class="table-list__meta" data-owner-region></span>
+      <span class="table-list__meta" data-due-region></span>
+      <span class="patient__action-ts">{{formatMoment updated_at "TIME_OR_DAY"}}</span>
+    </td>
```
Comment (luketlancaster) [inline src/js/views/patients/patient/data-events/action-item.hbs:9]: couplea nits to pick: does this need to be indented? And also should we throw in `{{~ remove_whitespace ~}}` like what we have in the `flow-item` template?

**Comment by paulfalgout** [comment]
The tests are failing consistently.. seems like a legit issue.

**Comment by LisaManresa** [comment]
Yup, I think I got them resolved now. Running coverage locally and checking the coverage report now.

File: test/integration/admin/programs/sidebar/flow-sidebar.js [hunk 6]
```diff
@@ -162,7 +162,13 @@ context('flow sidebar', function() {
 
         return fx;
       })
-      .routeProgramFlowActions(_.identity, '1')
+      .routeProgramFlowActions(fx => {
```
Comment (paulfalgout) [inline test/integration/admin/programs/sidebar/flow-sidebar.js:165]: 👍 ah flake.  good catch

File: test/support/api/flows.js [hunk 7]
```diff
@@ -1,5 +1,55 @@
 import _ from 'underscore';
-import { getResource, getRelationship } from 'helpers/json-api';
+import { getResource, getIncluded, getRelationship } from 'helpers/json-api';
+
+Cypress.Commands.add('routePatientFlows', (mutator = _.identity, patientId) => {
```
Comment (paulfalgout) [inline test/support/api/flows.js:4]: it probably does make sense to create some sort of data generator function that these two can share at some point as you were moving towards earlier.. but we can circle back around.

**Review by paulfalgout — APPROVED** [review]

**Pull request merged**